tests/integration_tests/functional/test_logging.py

```diff
@@ -44,16 +44,17 @@ def check_log_message_format(log_str, instance_id, level, show_level,
     Parse the string representing the logs and look for the parts
     that should be there.
     The log line should look lie this:
-         YYYY-MM-DDTHH:MM:SS.NNNNNNNNN [ID:LEVEL:FILE:LINE] MESSAGE
+         YYYY-MM-DDTHH:MM:SS.NNNNNNNNN [ID:THREAD:LEVEL:FILE:LINE] MESSAGE
     where LEVEL and FILE:LINE are both optional.
-    e.g.:
-    `2018-09-09T12:52:00.123456789 [MYID:WARN:/path/to/file.rs:52] warning`
+    e.g. with THREAD as "t":
+    `2018-09-09T12:52:00.123456789 [MYID:t:WARN:/path/to/file.rs:52] warning`
```

Comment:
**t** could be replaced with "thread-name"/"threadname" and `with THREAD as "t"` removed, since we also don't specify a set value for the other parameters beforehand.

Reply:
Good point, however adding either of those strings pushes line 50 above the 80-character line limit. Is it okay to break the example logger string on line 50 into two lines?

Reply:
Good observation. In this case, keeping the format you decided in the first place is a way to go. I'd rather see the log on one line if we can keep at that. I would modify it a little as "with THREAD NAME as **TN**" and it should look good in my opinion.

Reply:
Agreed! "TN" is more readable